packages/x-date-pickers/src/DatePicker/DatePickerToolbar.tsx

```diff
@@ -53,6 +53,15 @@ type DatePickerToolbarComponent = (<TDate>(
   props: DatePickerToolbarProps<TDate> & React.RefAttributes<HTMLDivElement>,
 ) => React.JSX.Element) & { propTypes?: any };
 
+/**
+ * Demos:
+ *
+ * - [DatePicker](https://mui.com/x/react-date-pickers/date-picker/)
```

Comment:
To make this completely correct we probably ought to add this component to the list of `components` on the `date-picker.md` 🤔 
```suggestion
 * - [DatePicker](https://mui.com/x/react-date-pickers/date-picker/)
 * - [Custom components](https://mui.com/x/react-date-pickers/custom-components/)
```